test/support/src/vfs_helpers.h

```diff
@@ -254,6 +257,9 @@ class SupportedFsS3 : public SupportedFs {
 
   /** If the filesystem is accessed via REST. */
   bool rest_;
+
+  /** True if the REST server is legacy, false if using 3.0 REST */
+  bool legacy_rest_;
```

Comment:
Is this used anywhere or just a leftover?

Reply:
Yep leftover from a previous commit, removed